### DISCUSSION TITLE
Como submeter sua palestra

### DISCUSSION THREAD

**Discussion by dearrudam** [soujava/comunidade#1] [Announcements]
Queremos que todas as pessoas possam compartilhar suas experiencias e aprendizados.

Para isso, tentamos montar uma forma simples para que possam submeter suas palestras para nossa comunidade:

### Passo 3. Aguarde o retorno de nossos colaboradores;

Muito obrigado por submeter sua palestra! :tada: :yellow_heart:

Caso tenham alguma dúvida, usem o espaço de comentários dessa issue para marcar os organizadores e saná-las.

**Comment by lari-nunes** [comment]
foi demais!
